media-sound/teamspeak-server-bin/files/ts3server-bin

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+BASEDIR="/opt/teamspeak3-server"
+
+export LD_LIBRARY_PATH="${BASEDIR}${LD_LIBRARY_PATH:+:}${LD_LIBRARY_PATH}"
+
+# work around DT_RPATH (.) security issue by chdir'ing into expected $LD_LIBRARY_PATH
+cd "${BASEDIR}"
```

Comment:
How about `&&` here (and drop the empty line below), so that it doesn't try to `exec` when `cd` fails?

Reply:
Sound fine for me. Will update this.